PolyEngine/Engine/Src/ConfigBase.hpp

```diff
@@ -9,7 +9,8 @@ namespace Poly
 	{
 		ENGINE,
 		GAME,
-		LOCAL
+		LOCAL,
+		RUNTIME_DIR
 	};
```

Comment:
We could use some comments here 😛 I'm assuming `RUNTIME_DIR` is `.`, `ENGINE` is some engine-specific config location (why do we need that?), while `GAME` is accordingly game-specific... What's `LOCAL`?

Reply:
True,
ENGINE will contain all engine configuration
GAME contains all game configs
LOCAL is all user configs stored in platform specific location
RUNTIME_DIR is what you said.

This enum will be probably merged with eResourceSource

Reply:
So why do we need separate directories for engine and game configuration files?
How is `LOCAL` different from `ENGINE` and `GAME`? Is it a parent directory? If so why do we need an explicit enumeration for it?